Data/Sys/GameSettings/R7P.ini

```diff
@@ -2,6 +2,7 @@
 
 [Core]
 # Values set here will override the main Dolphin settings.
+SkipIdle = False # Speedup
```

Comment:
*Very* unlikely. I guess you want to disable the "SyncOnIdleSkippingHack". Idle skipping is a safe enhancement, and I still haven't got why we allow to disable it at all.